server/src/main/java/app/cash/quickjs/QuickJs.java

```diff
@@ -0,0 +1,66 @@
+package app.cash.quickjs;
+
+import org.mozilla.javascript.ConsString;
+import org.mozilla.javascript.NativeArray;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import java.io.Closeable;
+
+public final class QuickJs implements Closeable {
+    private ScriptEngine engine;
+
+    public static QuickJs create() {
+        return new QuickJs(new ScriptEngineManager());
+    }
+
+    public QuickJs(ScriptEngineManager manager) {
+        this.engine = manager.getEngineByName("rhino");
+    }
+
+    public Object evaluate(String script, String fileName) {
+        return this.evaluate(script);
+    }
+
+    public Object evaluate(String script) {
+        try {
+            Object value = engine.eval(script);
+            return translateType(value);
+        } catch (Exception exception) {
+            exception.printStackTrace();
+            throw new QuickJsException(exception.getMessage());
```

Comment:
No newed to print the error either, the server will handle that
```suggestion
        } catch (Exception exception) {
            throw new QuickJsException(exception.getMessage());
```